.github/workflows/test.js.yml

```diff
@@ -0,0 +1,84 @@
+name: Run yarn test
+
+on:
+  push:
+    branches-ignore: 
+      - 'master'
+      - 'production'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    services:
+      postgres:
+        image: postgis/postgis:12-3.0
+        env:
+          POSTGRES_USER: arena
+          POSTGRES_PASSWORD: arena
+          POSTGRES_DB: of-arena-test
+        ports:
+          - 5444:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+          
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js 14.15.0
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.15.0'
+    - name: install browser dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libwoff1 \
+                      libopus0 \
+                      libwebp6 \
+                      libwebpdemux2 \
+                      libenchant1c2a \
+                      libgudev-1.0-0 \
+                      libsecret-1-0 \
+                      libhyphen0 \
+                      libgdk-pixbuf2.0-0 \
+                      libegl1 \
+                      libnotify4 \
+                      libxslt1.1 \
+                      libevent-2.1-6 \
+                      libgles2 \
+                      libgl1 \
+                      libegl1 \
+                      libvpx5 \
+                      libnss3 \
+                      libxss1 \
+                      libasound2 \
+                      libdbus-glib-1-2 \
+                      libxt6
+    - run: npm install -g chromium --unsafe-perm
+    - run: npm install pm2 -g
+    - run: cp test/db/migrations/*.js server/db/migration/public/migrations
+    - run: cp test/db/migrations/sqls/*.sql server/db/migration/public/migrations/sqls
+    - run: yarn
+    - run: yarn build:server:dev
+    - run: yarn build-dev
+    - run: ln -s dist/server.js .
+    - name: run server
+      run: exec pm2-runtime server.js &
+      env:
+        PGHOST: localhost
+        PGPORT: 5444
+        PGDATABASE: of-arena-test
+        PGUSER: arena
+        PGPASSWORD: arena
+        SESSION_ID_COOKIE_SECRET: my-cookie-secret-key
```

Comment:
Should we use github variables?

Reply:
yep I thinkt that should be a better way to set env vars

Reply:
You can use .env file or use github secrets/env vars